lib/utils.js

```diff
@@ -56,9 +56,10 @@ function renderTemplate({template, compileOptions, helpers, data}) {
   function addTemplateHelpersToData(templateData) {
     templateData.helpers = {};
     forOwn(helpers, (method, name) => {
-      data.helpers[name] = () => {
-        return (value) => {
-          return method(hogan.compile(value, compileOptions).render(templateData));
+      templateData.helpers[name] = function() {
+        return (text) => {
+          var render = (value) => hogan.compile(value, compileOptions).render(this);
```

Comment:
It's `method` which is `.call`ed, not `render`. Both need `this`, `render` can get it with the fat arraow, but I believe we need to use `.call` for `method`.